package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "university",
+  "version": "0.0.1",
+  "description": "Viget / Hapijs University",
+  "main": "lib/index.js",
+  "engines": {
+    "node": "5.3.0"
+  },
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "start": "node .",
+    "test": "lab -t 100",
+    "test-cov-html": "lab -r html -o coverage.html"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vigetlabs/university.git"
+  },
+  "author": "Mike Ackerman",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/vigetlabs/university/issues"
+  },
+  "homepage": "https://github.com/vigetlabs/university#readme",
+  "dependencies": {
+    "hapi": "^11.1.4"
+  },
+  "devDependencies": {
+    "code": "^2.1.0",
+    "lab": "^8.0.1"
+  }
```

Comment:
We've started using fixed dependencies because package maintainers love to release minor breaking updates. What do you think about killing the `^` hats here?